Journals_API.ipynb

```diff
@@ -0,0 +1,506 @@
+            "            \"name\": \"Physics &amp; Astronomy International Journal\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"RadA\",\n",
+            "            \"name\": \"Radio Astronomy: Journal of the Society of Amateur Radio Astronomers,\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"2010AIPC.1283\",\n",
+            "            \"name\": \"Mathematics and Astronomy: A Joint Long Journey\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"IrAJ\",\n",
+            "            \"name\": \"Irish Astronomical Journal\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"IrAJS\",\n",
+            "            \"name\": \"Irish Astronomical Journal Supplement\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"RoAJ\",\n",
+            "            \"name\": \"Romanian Astronomical Journal\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"AEdJ\",\n",
+            "            \"name\": \"Astronomy Education Journal\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"SerAJ\",\n",
+            "            \"name\": \"Serbian Astronomical Journal\"\n",
+            "        },\n",
+            "        {\n",
+            "            \"bibstem\": \"AzAJ\",\n",
+            "            \"name\": \"Azerbaijani Astronomical Journal\"\n",
+            "        }\n",
+            "    ]\n",
+            "}\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100  1151  100  1151    0     0   3905      0 --:--:-- --:--:-- --:--:--  4052\n"
+          ]
+        }
+      ],
+      "source": [
+        "%%bash\n",
+        "token=\"your-token-here\"\n",
+        "curl -H \"Authorization: Bearer $token\" https:\/\/api.adsabs.harvard.edu\/v1\/journals\/Journal\/Astronom%20Jour | python3 -m json.tool\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### `/Refsource`\n",
+        "Generate a list of citation data sources on a per volume basis for a given `<bibstem>`\n",
+        "\n",
+        "*Return data*: output is JSON\n",
+        "* `refsource` (dict): list of volume, year, and reference sources for in ADS holdings for a given bibstem\n",
+        "  *  `bibstem` (text): ADS bibstem for a given Title\n",
+        "  *  `volumes` (list, dict): refsource data for individual volumes of Title\n",
```

Comment:
I'd define what "refsource" is here

Reply:
Modified text: `refsources` (dict): number of papers with reference data originating from each `<data source>` in the volume\n",